modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/itemselector/plugin.js

```diff
@@ -452,22 +452,17 @@
 							if (typeof callback === 'function') {
 								callback(imageSrc, selectedItem);
 							} else {
-								var elementOuterHtml =
-									'<img src="' + imageSrc + '">';
-
-								editor.insertHtml(elementOuterHtml);
-
 								if (instance._isEmptySelection(editor)) {
 									if (IE9AndLater) {
 										var usingAlloyEditor =
```

Comment:
We should also update this to be consistent with `typeof editor.window.$.AlloyEditor == 'undefined';`